go/teams/loader.go

```diff
@@ -724,6 +724,17 @@ func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (
 		}
 	}
 
+	if ret == nil {
+		return nil, fmt.Errorf("team loader fault: got nil from load2")
+	}
+
+	encKID, gen, role, err := l.hiddenPackageGetter(mctx, arg.teamID, ret, arg.me)()
+	if err != nil {
+		return nil, err
+	}
+	// update the hidden package once ret != nil
```

Comment:
can you say why we do this? seems like we really only need to do this for the hidden bot case, wanting to skip seedchecks...